lib/thrift/parser/conversions.ex

```diff
@@ -6,6 +6,16 @@ defmodule Thrift.Parser.Conversions do
     List.to_atom(l)
   end
 
+  def atomic_snake(nil), do: nil
```

Comment:
This could use a little `@doc` and/or `@spec` to describe its behavior. It's only semi-obvious from its name. ⚛️  🐍